docs/guide-ja/glossary.md

```diff
@@ -1,73 +1,73 @@
 # A
 
-## alias �G�C���A�X
+## alias エイリアス
 
-�G�C���A�X�́A�N���X��f�B���N�g��������߂� Yii �ɂ��ėp�����镶����ł��B�Ⴆ�� `@app/vendor`�B
+エイリアスは、クラスやディレクトリを示すために Yii によって用いられる文字列です。例えば `@app/vendor`。
 
-## application �A�v���P�[�V����
+## application アプリケーション
 
-�A�v���P�[�V������ HTTP ���N�G�X�g�����̂��߂̒��S�I�ȃI�u�W�F�N�g�ł��B
-���ɕ�鐔�����̃R���|�[�l���g�ɂ��āA���N�G�X�g�������擾���A�X�ɏ�����i�߂邽�߂ɂ����K�؂ȃR���g���[���ɑ��v���܂��B
+アプリケーションは HTTP リクエスト処理のための中心的なオブジェクトです。
+内部に抱える数多くのコンポーネントによって、リクエストから情報を取得し、更に処理を進めるためにそれを適切なコントローラに送致します。
 
-�A�v���P�[�V�����E�I�u�W�F�N�g�̓G���g���E�X�N���v�g�ɂ��ăV���O���g���Ƃ��ăC���X�^���X������܂��B
-�A�v���P�[�V�����E�V���O���g���͂ǂ̏ꏊ����ł� `\Yii::$app` �Ƃ��ăA�N�Z�X�\�ł��B
+アプリケーション・オブジェクトはエントリ・スクリプトによってシングルトンとしてインスタンス化されます。
+アプリケーション・シングルトンはどの場所からでも `\Yii::$app` としてアクセス可能です。
 
-## assets �A�Z�b�g
+## assets アセット
 
-�A�Z�b�g�Ƃ̓��\�[�X�E�t�@�C����w��������̂ł��B
-�T�^�I�ɂ� JavaScript �܂��� CSS ��܂ރt�@�C���ł����AHTTP �ɂ��ăA�N�Z�X�\�Ȃ�̂Ȃ牽�ł�\���܂���Bt or CSS code but can be anything else that is accessed via HTTP.
+アセットとはリソース・ファイルを指し示すものです。
+典型的には JavaScript または CSS を含むファイルですが、HTTP によってアクセス可能なものなら何でも構いません。t or CSS code but can be anything else that is accessed via HTTP.
```

Comment:
Not translated?

Reply:
Ohhhh, my bad. I forgot to erase the garbage.

Reply:
I've just fixed it.